api/can_api_v2_definition.py

```diff
@@ -61,6 +61,9 @@ class Metrics(base_model.APIBaseModel):
         ...,
         description="Ratio of people who test positive calculated using a 7-day rolling average.",
     )
+    testPositivityRatioMethod: Optional[
```

Comment:
My first thought is that this would be better served as `testPositivityRatioDetails` with the `method` being the only item inside.  

I think it'll make it more future proof.  If we ever add another metadata option (don't see one immediately, but wouldn't shock me) it will make it easier to make a non breaking change.  In that case, I would prefer to have it in its own object rather than cluttering the top level metrics space.  Additionally it copies an existing pattern of `icuHeadroomDetails`.

Reply:
testPositivityRatioDetails =
```
{
  source: 'CMS'
}
```